network/p2p/unicast/protocol.go

```diff
@@ -0,0 +1,72 @@
+package unicast
```

Comment:
Spoiler alert: not new logic, mostly logic migration from `p2p/protocol.go` -> `p2p/unicast/protocol.go`